lib/themes/dosomething/paraneue_dosomething/js/components/Slideshow.js

```diff
@@ -20,9 +21,16 @@ class Slideshow extends React.Component {
   }
 
   render() {
+    const user = this.props.user;
+    const campaign = this.props.campaign;
+
+    const slides = this.props.slides.map((slide, index) => {
+      return React.createElement(slide, {key: index, campaign: campaign, user: user});
+    });
+
     return (
       <div className="slideshow">
-        {this.props.slides[this.state.activeSlide]}
+        {slides}
```

Comment:
talked with @weerd , came to the suggestion we can keep `{slides}` in the render loop so all of the React elements are available on the page for CSS styling. however the `const slides` should be moved into the constructor and set as a property of the Slideshow, rather than re-creating every slide on every `render()`. @DFurnes thoughts?